packages/oid4vci-issuer/src/functions.ts

```diff
@@ -239,6 +243,20 @@ export async function getCredentialSignerCallback(
           _sd: credential['_sd'],
         }
       }
+
+      if (contextHasPlugin<IStatusListPlugin>(context, 'slAddStatusToCredential') && sdJwtPayload.status && sdJwtPayload.status.status_list) {
+        // Add status list if enabled (and when the input has a credentialStatus object (can be empty))
+        const credentialStatusVC = await context.agent.slAddStatusToSdJwtCredential({ credential: sdJwtPayload })
+        if (sdJwtPayload.status?.status_list?.idx) {
+          if (!credentialStatusVC.status || !credentialStatusVC.status.status_list) {
+            // TODO check, looks like sdJwtPayload and credentialStatusVC is the same
```

Comment:
Please create a ticket for it or solve it in this ticket

Reply:
No really important. It is just a remark that 2 types look the same. Not every TODO has to result in a ticket